tests/parallel/parallel_test.C

```diff
@@ -145,7 +145,7 @@ public:
 
         for (unsigned int i=0; i<src.size(); i++)
           src[i] = i;
-        for (unsigned int i=0; i<src.size(); i++)
+        for (unsigned int i=0; i<TestCommWorld->size(); i++)
```

Comment:
Buggy tests are definitely better than no tests.  We probably ought to have Civet running all our tests in parallel, though; that's a common enough use case to be worth catching at the source.